icc-api/model/ClassificationDto.ts

```diff
@@ -57,8 +57,4 @@ export class ClassificationDto {
   encryptionKeys?: { [key: string]: Array<models.DelegationDto> }
 
   templateId?: string
-
```

Comment:
@cebamps I guess this can not be accepted as it would break the 3rd level

Reply:
Indeed, Medispring will fail to compile in prod mode.